### PR TITLE
Input Simulation Port: Camera

### PR DESCRIPTION
**Pull request by FejZa**
## Overview

This PR builds on top of #46 and adds default profiles for camera input simulation as well as an update to the the default input simulation data providers profile to register the camera provider. The related Core PR is https://github.com/XRTK/XRTK-Core/pull/227

## Changes:

- Add default camera provider profile
- Add camera provider to default providers for simulation

### CODE REVIEW

File: XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/Simulation/DefaultMixedRealityInputSimulationDataProvidersProfile.asset [hunk 1]
```diff
@@ -13,4 +13,11 @@ MonoBehaviour:
   m_Name: DefaultMixedRealityInputSimulationDataProvidersProfile
   m_EditorClassIdentifier: 
   isCustomProfile: 0
-  registeredInputSimulationDataProviders: []
+  registeredInputSimulationDataProviders:
+  - dataProviderType:
+      reference: XRTK.Providers.InputSystem.Simulation.CameraSimulationDataProvider,
+        XRTK
+    dataProviderName: Camera Input Simulation Data Provider
```
Comment (StephenHodgson) [inline XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/Simulation/DefaultMixedRealityInputSimulationDataProvidersProfile.asset]: What exactly is `Camera Input`?

Comment (FejZa) [reply XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/Simulation/DefaultMixedRealityInputSimulationDataProvidersProfile.asset]: The naming is maybe not perfect. In MRTK it was called "ManualCameraControl". It's basically not input but simulated head / camera movement using keyboard and mouse. So we are not simulating some sort of camera input / video stream or whatever but we are simulating movement of the camera here.

Comment (FejZa) [reply XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/Simulation/DefaultMixedRealityInputSimulationDataProvidersProfile.asset]: Does that help?

Comment (SimonDarksideJ) [reply XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/Simulation/DefaultMixedRealityInputSimulationDataProvidersProfile.asset]: I'd agree the naming probably doesn't help, as in the scenario you describe, the camera is an Actor, not an Input, as in, we are taking input to affect the camera.

Are we likely over complicating this taking directly what MRTK implemented, is this simply not just a special controller which can have a camera attached (remembering, Unity can have multiple cameras in a scene)

Does that make sense?

Comment (StephenHodgson) [reply XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/Simulation/DefaultMixedRealityInputSimulationDataProvidersProfile.asset]: If the camera is receiving input from some input data provider then this should be a handler not a service.

**Pull request closed without merging**